src/components/AudioPlayer/AudioPlayer.container.js

```diff
@@ -1,35 +1,51 @@
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 
-import { setIsPlaying } from 'store/actions/playerActions';
+import { setIsPlaying, playNextSong } from 'store/actions/playerActions';
 
 import AudioPlayerComponent from './AudioPlayer.component';
 
 const AudioPlayerContainer = () => {
-  const autoPlay = useSelector((state) => state.settings.get('autoPlay'));
   const songLink = useSelector((state) => state.player.get('songLink'));
   const dispatch = useDispatch();
 
-  const audioRef = React.createRef();
+  const audioRef = new Audio(songLink);
```

Comment:
why create it?, cant we pass that from somewhere? (talking about workspace in general) not sure.